root/help.md

```diff
@@ -251,7 +255,7 @@ You find plenty of well-known tools within this container. Here comes the full l
 You should invoke this container using `atomic` command:
 
 ```
-$ atomic run tools
+$ atomic run modularitycontainers/tools
```

Comment:
Perhaps you want to template this as well as `{{ spec.image_name }}`?

Reply:
`help/help.md` is the template, `root/help.md` is the rendered help file

This is what I struggled with: should we also include upstream render in the repo? My opinion is: yes, we should:
 * outsiders won't be confused
 * we can easily test upstream in the CI

Reply:
Oh, I haven't noticed that. Personally, I'd rather not include any rendered artifacts, but I guess it's a matter of taste.

Reply:
should we talk about this?